src/pages/SplashPage.vue

```diff
@@ -26,15 +26,62 @@
 </template>
 
 <script setup>
+import { Notify } from 'quasar';
+import { useI18n } from 'vue-i18n';
 import { useRoute, useRouter } from 'vue-router';
 import { onMounted } from 'vue';
 import { initPlugins } from 'src/composables/PluginManager';
 import PluginEvent from 'src/composables/events/PluginEvent';
+import { getUserSessionToken, login, getUserInfo } from 'src/composables/UserAuthentication';
+import { useUserStore } from 'src/stores/UserStore';
 
+const { t } = useI18n();
 const route = useRoute();
 const router = useRouter();
 
+/**
+ * Initialize the user for the appplication, if needed.
+ * If the backend is activated in the configuration, it will check
+ * that the session token is present or not. If not, it will try to login the user.
+ * If already connected, it will just retrieve the user info.
+ */
+async function initUser() {
+  if (process.env.IFBACKEND) {
+    // TODO: Add another if to check if the session token is expired
+    if (!getUserSessionToken()) {
```

Comment:
same:

if (!getUserSettionToken) {
  return login(...)...;
}

Reply:
I do not agree with you on that one. Because it makes the code less readable.

Moreover I can't really separate these 2 if because they are already 2 separate condition in which case we do not have the same behavior.